docs/openapi/paths/activity-reports/approvers.yaml

```diff
@@ -0,0 +1,14 @@
+get:
+  tags:
+    - activity-reports
+  operationId: getApproves
+  description: Gets all approvers for any region the current user has write permissions
+  responses:
+    200:
+      description: All approvers for any region the user has write permissions
+      content:
+        application/json:
+          schema:
+            type: array
+            items:
+              $ref: '../../index.yaml#/components/schemas/approvingUser'
```

Comment:
I'd like for a better solution for these relative links, seems like they could potentially get pretty unwieldy